pkg/eraser/eraser.go

```diff
@@ -151,6 +162,45 @@ func removeImages(c Client, targetImages []string) error {
 	return nil
 }
 
+func isExcluded(img string, idToTagListMap map[string][]string) bool {
+	// check if img excluded by digest
+	if _, contains := excluded[img]; contains {
+		return true
+	}
+
+	// check if img excluded by name
+	if len(idToTagListMap[img]) > 0 {
+		if _, contains := excluded[idToTagListMap[img][0]]; contains {
+			return true
+		}
+	}
```

Comment:
An image can have multiple names associated with it. We need to consider all of them, not just the first, or we'll run afoul of the alias bug we fixed earlier.

```suggestion
	for _, imgName := range idToTagListMap[img] {
	    if _, contains := excluded[imgName]; contains {
	        return true
	    }
	}
```